### ISSUE TITLE
Ability to append custom text to an issue's body

### ISSUE THREAD

**Issue by php-coder** [yegor256/0pdd#164]
Extracted from https://github.com/yegor256/0pdd/issues/148#issuecomment-348781406:

I'd like to be able to add an additional information to the body of every issue that is being created by bot. It should not just allow me to append a text but also support simple templating: commit, original issue number, and, probably, author should be supported.

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#164](https://github.com/yegor256/0pdd/issues/164) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#164](https://github.com/yegor256/0pdd/issues/164) assigned to tedtoer. The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@tedtoer[/z](https://www.0crat.com/u/tedtoer) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@tedtoer[/z](https://www.0crat.com/u/tedtoer) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
The user @tedtoer[/z](https://www.0crat.com/u/tedtoer) resigned from [#164](https://github.com/yegor256/0pdd/issues/164), please stop working

**Comment by 0crat** [comment]
The job [#164](https://github.com/yegor256/0pdd/issues/164) assigned to @rok-povsic[/z](https://www.0crat.com/u/rok-povsic), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/451714310). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by rok-povsic** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/0pdd/issues/164#issuecomment-367626663))

@rok-povsic The user @rok-povsic[/z](https://www.0crat.com/u/rok-povsic) resigned from [#164](https://github.com/yegor256/0pdd/issues/164), please stop working

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://datum.zerocracy.com/pages/policy.html#6): -15 points just awarded to @rok-povsic[/z](https://www.0crat.com/u/rok-povsic), total is [+105](https://www.0crat.com/u/rok-povsic)

**Comment by 0crat** [comment]
The job [#164](https://github.com/yegor256/0pdd/issues/164) assigned to @prondzyn[/z](https://www.0crat.com/u/prondzyn), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/541246535). The budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
The user @prondzyn[/z](https://www.0crat.com/u/prondzyn) resigned from [#164](https://github.com/yegor256/0pdd/issues/164), please stop working. Reason for job resignation: It is older than 10 days, see [§8](http://www.zerocracy.com/policy.html#8)

**Comment by 0crat** [comment]
Resigned on delay, see [§8](http://www.zerocracy.com/policy.html#8): -30 points just awarded to @prondzyn[/z](https://www.0crat.com/u/prondzyn)

**Comment by izrik** [comment]
@php-coder Please clarify: when you say "add an additional comment", do you mean including additional text in the description of the issue? Or a separate comment, just like the comment I'm writing now?
Also, would this be the same thing added to every issue that gets opened? 0pdd will already include the text of the puzzle in the issue description. Is that not sufficient?

**Comment by php-coder** [comment]
> Please clarify: when you say "add an additional comment", do you mean including additional text in the description of the issue?

Yes, in the description. I've updated description of this issue to make it clearer.

**Comment by 0crat** [comment]
@izrik[/z](https://www.0crat.com/u/izrik) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by izrik** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/0pdd/issues/164#issuecomment-382608800))

@izrik This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#164](https://github.com/yegor256/0pdd/issues/164) is [in scope](http://www.zerocracy.com/policy.html#14) for 3mon
  * The role is `DEV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @php-coder[/z](https://www.0crat.com/u/php-coder): This user reported the ticket
    * @tedtoer[/z](https://www.0crat.com/u/tedtoer): User was resigned from the ticket
    * @rok-povsic[/z](https://www.0crat.com/u/rok-povsic): User was resigned from the ticket
    * @prondzyn[/z](https://www.0crat.com/u/prondzyn): User was resigned from the ticket
    * @izrik[/z](https://www.0crat.com/u/izrik): User was resigned from the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3T46CUJJ?q=%7Bjob%3A%27gh%3Ayegor256%2F0pdd%23164%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
The job [#164](https://github.com/yegor256/0pdd/issues/164) assigned to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel), here is [why](https://www.0crat.com/footprint/C3T46CUJJ/1130934871); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by SilasReinagel** [comment]
@0crat wait - Blocked by #200

**Comment by 0crat** [comment]
> @0crat wait - Blocked by #200  ([here](https://github.com/yegor256/0pdd/issues/164#issuecomment-384503569))

@SilasReinagel The impediment for [#164](https://github.com/yegor256/0pdd/issues/164) was registered successfully by @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)
